src/compounds/legacy-product-table/src/index.tsx

```diff
@@ -217,51 +351,91 @@ interface DataCellProps extends React.HTMLAttributes<HTMLDivElement> {
   borderBottomColor?: string
   color?: string
   label?: string
+  isWireFrame?: boolean
 }
 
 export const DataCell: React.FC<DataCellProps> = ({
   children,
   backgroundColor,
   borderBottomColor,
   color,
-  label
+  label,
+  isWireFrame = false
 }) => {
+  const wireFrameLabelStyles = {
+    width: '100%',
+    maxWidth: ['134px', '84px'],
```

Comment:
these numbers feel a bit magic, where do they come from, can we get anything from the theme?

Reply:
Turned them into percentages according to the design